wrangler-proto/src/main/java/co/cask/wrangler/proto/connection/Connection.java

```diff
@@ -37,14 +37,23 @@
   // Time in second - when it was last updated.
   private final long updated;
 
+  // Determines if the connection is configured automatically
+  private final boolean builtIn;
```

Comment:
How about preconfiguredConnection as the parameter name?

Reply:
I like it. perhaps just ``preconfigured`` though, since this is the Connection class already?

Reply:
addressed